### PR TITLE
[nodejs] fix ranking csv bug

### PR DESCRIPTION
**Pull request by chejennifer**
fix bug where ranking table csv is in the wrong order

updated response for family earnings in california counties
https://paste.googleplex.com/6309252029743104

### CODE REVIEW

File: static/nodejs_server/ranking_tile.ts [hunk 1]
```diff
@@ -107,8 +107,11 @@ function getRankingUnitResult(
   useChartUrl: boolean,
   apiRoot: string
 ): TileResult {
+  const points = isHighest
+    ? rankingGroup.points.slice().reverse()
```
Comment (pradh) [inline static/nodejs_server/ranking_tile.ts:111]: Will this sort by RankingPoint.value?

**Review by pradh — APPROVED** [review]
Thanks!

File: static/nodejs_server/ranking_tile.ts (hunk 1 above)
Comment (chejennifer) [reply static/nodejs_server/ranking_tile.ts:111]: yes, rankingGroup.points is already sorted by value, but from lowest to highest

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**